scripts/setup-project.sh

```diff
@@ -159,6 +159,13 @@ create_threadx_projects() {
       project_ide_dir='mplab/'
       libs="same54_lib filex common_hardware_code "
       ;;
+    ck-rx65n)
+      echo Downloading Azure_RTOS_6...
+      wget -q -O azrtos.zip https://saleshosted.z13.web.core.windows.net/sdk/AzureRTOS/Azure_RTOS_6.1_RX65N_Cloud_Kit_E2Studio_GNURX_Samples_2022_05_25.zip
```

Comment:
Need to use ${FETCH} rather than wget here.

Reply:
See https://github.com/avnet-iotconnect/iotc-azurertos-sdk/pull/48

Reply:
I'd made that change in https://github.com/avnet-iotconnect/iotc-azurertos-sdk/pull/47/files#diff-42bd5a8d71dda5434de8eb708560bad0c8f448068c38766613305898b27f35f8 which is branched from this branch, but not in here. Could merge PR #48 and rebase PR#47 on top of that if that is desired